docs/core/testing/microsoft-testing-platform-intro.md

```diff
@@ -45,10 +45,13 @@ The main driving factors for the evolution of the new testing platform are detai
 
 ## Run and debug tests
 
-`Microsoft.Testing.Platform` test projects are built as executables that can be run (or debugged) directly. There's no extra test running console or command. The app exits with a nonzero exit code if there's an error, as typical with most executables. For more information on the known exit codes, see [Microsoft.Testing.Platform exit codes](unit-testing-platform-exit-codes.md).
+`Microsoft.Testing.Platform` test projects are built as executables that can be run (or debugged) directly. There's no extra test running console or command. The app exits with a nonzero exit code if there's an error, as typical with most executables. For more information on the known exit codes, see [Microsoft.Testing.Platform exit codes](microsoft-testing-platform-exit-codes.md).
+
+> [!TIP]
+> You can ignore a specific exit code using the `--ignore-exit-code` command line option. You can also set command line options that apply to a specific test project in the project file using the `TestingPlatformCommandLineArguments` MSBuild property. One common use case is for test projects that have all the tests ignored, which will normally exit with exit code 8 (the test session ran zero tests). In this scenario, you can add `<TestingPlatformCommandLineArguments>$(TestingPlatformCommandLineArguments) --ignore-exit-code 8</TestingPlatformCommandLineArguments>` to your project file.
```

Comment:
@RussKie Does the tip in this page look sufficient to address the doc concern you have in https://github.com/microsoft/testfx/issues/5146 ? Please let me know if we can do better here.

Reply:
That's definitely an improvement, thank you.

Few suggestions:

* I'd link every piece to respective doc page (like I did in https://github.com/microsoft/testfx/issues/5146#issuecomment-2693056503) - this way readers can navigate to individual docs directly and learn more info.

* I'd also suggest formatting the inline snippet as tripple-backtick code snippet (like other snippets) - this will increase the prominence of it.

Reply:
Thanks! Done in the last commit. Hopefully it's easier to find the information now?
Note: I'm still planning to do more doc updates in follow-up PRs to improve the overall quality, as well as discoverability, of information in docs.

Reply:
Whether it's easier to find or not is an open question :) When a test fails without any clear indication it's still hard to know what to look for. But it's definitely an improvement, and the chances of a lost-developer discovering this information IMO increased.
Thank you

Reply:
> When a test fails without any clear indication it's still hard to know what to look for.

I'm hoping that we can change that in product-level (in `dotnet test`) so that it clearly indicates the exit code with a documentation link.